v1/backend_users/backend_users/backend_users.csproj

```diff
@@ -1,19 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Ik loop tegen deze foutmelding aan wanneer ik deze branch build (zonder cache, `docker-compose build --no-cache`)

```
error NU1202: Package Npgsql.EntityFrameworkCore.PostgreSQL 5.0.1 is not compatible with net50 (.NETFramework,Version=v5.0). Package Npgsql.EntityFrameworkCore.PostgreSQL 5.0.1 supports: netstandard2.1 (.NETStandard,Version=v2.1)
/app/backend_users.csproj : error NU1202: Package Microsoft.EntityFrameworkCore 5.0.1 is not compatible with net50 (.NETFramework,Version=v5.0). Package Microsoft.EntityFrameworkCore 5.0.1 supports: netstandard2.1 (.NETStandard,Version=v2.1)
/app/backend_users.csproj : error NU1202: Package Microsoft.AspNetCore.App 2.2.8 is not compatible with net50 (.NETFramework,Version=v5.0). Package Microsoft.AspNetCore.App 2.2.8 supports: netcoreapp2.2 (.NETCoreApp,Version=v2.2)
  Restore failed in 2.34 sec for /app/backend_users.csproj.
  Restore completed in 2.87 sec for /app/backend_users.csproj.
ERROR: Service 'users' failed to build : The command '/bin/sh -c dotnet restore' returned a non-zero code: 1
```

Reply:
Denk je dat we nog iets moeten met deze warnings?
```/app/backend_users.csproj : warning NU1701: Package 'Microsoft.AspNet.WebApi.Client 4.0.20710' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project.
/app/backend_users.csproj : warning NU1701: Package 'Microsoft.AspNet.WebApi.Core 4.0.20710' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project.
/app/backend_users.csproj : warning NU1701: Package 'Microsoft.Net.Http 2.0.20710' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project.
/app/backend_users.csproj : warning NU1701: Package 'Swashbuckle.Core 5.6.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project.
```

Reply:
Ja ik maak wel even een notitie dat ik de depencies in de gaten houd en update wanneer er een update is.